src/pages/about.jsx

```diff
@@ -30,57 +36,144 @@ export const About = () => {
+            >
+              {Items.map((item, index) => {
+                return (
+                  <Grid item xs={12} sm={6} md={3} key={index}>
+                    {/* There are two conditions 
+                    1) for mobile
+                      a) showing default only two projects  -> (mobileActive && index < 2)
+                      b) if user wants to see more Projects -> (seeMore)
+                    2) for Desktop, show all projects       -> (!mobileActive) */}
+                    {((mobileActive && index < 2) ||
+                      seeMore ||
+                      !mobileActive) && (
+                      <AboutCard
+                        item={item}
+                        mobileActive={mobileActive}
+                        seeMore={seeMore}
+                      />
+                    )}
+                  </Grid>
+                )
+              })}
+            </Grid>
+          </Box>
+
+          {mobileActive && <Mobile setseeMore={setseeMore} seeMore={seeMore} />}
+
+          <Box mx={{ xs: 1, sm: 10, md: 20, lg: 40 }} my={13}>
+            <Typography
+              mt={1}
+              mx={1}
+              align="center"
+              variant="h5"
+              sx={{
+                fontFamily: 'Mukta',
+                fontWeight: '600',
+              }}
+            >
+              We Welcome Your Participation!
             </Typography>
-          </CardContent>
-        </Card>
-      </div>
-      <h2 className={aboutStyles.particiation}>
-        <strong>We Welcome Your Participation</strong>
-      </h2>
-      <p className={aboutStyles.join}>
-        {`You can join us anytime as a volunteer to our open-source software &
-        hardware projects.`}
-        <br></br>
-        <br></br>
-        {`If you'd like to host a hydrophone, do research, or incorporate
-        Orcasound into the educational or outreach efforts of your
-        organization,please reach out!`}
-      </p>
-      <div className={aboutStyles.button}>
-        <Button variant="outline-primary"> GET INVOLVED!</Button>
-      </div>
-    </div>
+
+            <Box my={3}>
+              <Typography
+                mx={1}
+                mt={3}
+                align="justify"
+                variant="body1"
+                gutterBottom
+              >
+                You can join us anytime as a volunteer to our open-source
+                software & hardware projects.
+              </Typography>
+              <Typography
+                mx={1}
+                mt={3}
+                align="justify"
+                variant="body1"
+                gutterBottom
+              >
+                If you'd like to host a hydrophone, do research, or incorporate
+                Orcasound into the educational or outreach efforts of your
+                organization,please reach out!
+              </Typography>
+            </Box>
+
+            <Box
+              my={3}
+              mt={4}
```

Comment:
Doesn't make sense to use both `my` and `mt`, even if it technically works, it's confusing